_shared/project/tox.ini

```diff
@@ -30,10 +30,13 @@ setenv =
     dev: NEW_RELIC_APP_NAME = {env:NEW_RELIC_APP_NAME:{{ cookiecutter.slug }}}
     dev: NEW_RELIC_ENVIRONMENT = {env:NEW_RELIC_ENVIRONMENT:dev}
 {% if cookiecutter.get("postgres") == "yes" %}
+    dev: ALEMBIC_CONFIG = {env:ALEMBIC_CONFIG:via/migrations/alembic.ini}
     SQLALCHEMY_SILENCE_UBER_WARNING=1
     dev: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/postgres}
-    tests: TEST_DATABASE_URL = {env:TEST_DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_tests}
-    functests: TEST_DATABASE_URL = {env:TEST_DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_functests}
+    tests: DATABASE_URL = {env:DATABASE_URL:postgresql://postgres@localhost:{{ cookiecutter['__postgres_port'] }}/{{ cookiecutter.package_name }}_tests}
```

Comment:
Having the envvar be called `DATABASE_URL` in dev and prod but `TEST_DATABASE_URL` in the tests was a pain because the code when being run by the tests would try to read `DATABASE_URL` and crash. Keeping the envvar name the same just makes things simpler